fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ParameterDAOImpl.java

```diff
@@ -216,6 +246,19 @@ public void addParameterNamesCacheCandidate(String parameterName, Integer parame
 
     }
 
+
+    protected  Integer getParameterNameIdFromCaches(String parameterName) {
+        // Get ParameterNameId from ParameterNameIdCache first.
+        Integer parameterNameId = ParameterNamesCache.getParameterNameId(parameterName);
+        // If no found, then get ParameterNameId from local newParameterNameIds in case this id is already in newParameterNameIds
```

Comment:
```suggestion
        // If not found, then get ParameterNameId from local newParameterNameIds in case this id is already in newParameterNameIds
```